pkgs/development/python-modules/onnx/default.nix

```diff
@@ -36,11 +33,11 @@ buildPythonPackage rec {
     numpy
     six
     typing-extensions
-  ] ++ lib.optional (pythonOlder "3.5") [ typing ];
+  ];
 
   checkInputs = [
+    pytestCheckHook
     pytestrunner
```

Comment:
This should be in `nativeBuildInputs` because it's required even if `doCheck = false`. Even better would be to patch the dependency out.

Reply:
Done, but what's the reason for patching it out (the tests do all run without it, so is it just unnecessary)?

Reply:
It's unnecessary and, if I remember, has caused problems in the past. It's also counterinuitive to have `pytestrunner` as an input even though we run the tests in a different way.